blog/_posts/2022-09-29-scala-survey-2022-announcement.md

```diff
@@ -9,9 +9,11 @@ title: Participate in the Scala Survey 2022!
 
 ## The Scala Team wants to hear from you!
 
-We want to make the language we all love as amazing as possible. We want to know which way the Scala community is moving and which parts of the Scala language you utilize the most. That’s why ask you to help by participating in the Scala Survey 2022.
+We want to make the language we all love as amazing as possible. We want to know which way the Scala community is moving and which parts of the Scala language you utilize the most. That’s why we invite you to help by participating in the Scala Survey 2022.
 
-Survey: [Scala Survey 2022](https://scala-survey-2022.virtuslab.com/)
+[Scala Survey 2022](https://scala-survey-2022.virtuslab.com/)
+
+The Survey will close on **21.10.2022**.
 
 ## About the survey
 
```

Comment:
The next line needs to be more persuasive for users to complete the survey, this should just say how long the survey takes

Reply:
can keep the part about it being anonymous also